### PR TITLE
Improve README for newcomers

### PR DESCRIPTION
**Pull request by temeddix**
* Ticket : #1

When I first saw this library, it wasn't certain if this library only provides JavaScript API, or is directly usable in Rust. I had to do some experiments to find out that it works in Rust code, and make an issue to understand how this system works.

This library is really powerful, and I believe that this library has the potential to become bigger like `proj` crate.

However, not all people who are doing geographic coordinate conversion are geo experts(including myself). These enhancements in the first guide will allow people seeking for coordinate conversion to adopt this library much more easily.

Hope this change can make into GitHub first page and crates.io :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,64 @@
...
+    "+proj=longlat +ellps=WGS84",
+    " +datum=WGS84 +no_defs"
+))
+.unwrap();
+
+let mut point_3d = (198236.3200000003, 453407.8560000006, 0.0);
+proj4rs::transform::transform(&from, &to, &mut point_3d).unwrap();
+
+// Note that WGS84 output from this library is in radians, not degrees.
+point_3d.0 = point_3d.0 * (180.0 / std::f64::consts::PI);
```
Comment (dmarteau) [inline README.md:56]: You sould use [f64::to_degrees()[https://doc.rust-lang.org/std/primitive.f64.html#method.to_degrees)

File: README.md [hunk 2]
```diff
@@ -1,22 +1,64 @@
...
+point_3d.1 = point_3d.1 * (180.0 / std::f64::consts::PI);
```
Comment (dmarteau) [inline README.md:57]: Same here,

**Review by dmarteau — CHANGES_REQUESTED** [review]
Great ! thank you very much !

I just pointed minor improvements in the examples

File: README.md (hunk 1 above)
Comment (temeddix) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (temeddix) [reply README.md]: This is also fixed

**Review by dmarteau — APPROVED** [review]
Thanks @temeddix

**Review by Gustry — APPROVED** [review]

**Comment by Gustry** [comment]
Thanks for your contribution.

I just fixed some typo (not related to your PR)
cd4a64d5b3547d07dea2ddb3847d8e5930d39cd9

**Pull request merged**